util/user.py

```diff
@@ -104,3 +105,16 @@ def name_from_id(ctx, user_id):
                                      genquery.AS_LIST, ctx):
         return row[0]
     return ''
+
+
+def number_of_connections(ctx):
+    """Get number of active connections from client user."""
+    connections = 0
+    try:
+        ips = subprocess.check_output(["ips", "-a"])
+        username = session_vars.get_map(ctx.rei)['client_user']['user_name']
+        connections = ips.count(username)
+    except Exception:
+        return 0
```

Comment:
I think it would be good to also log the exception here, since this is an unexpected situation.